playwright/tests/screenshots/browsers.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { test, expect, type Page } from '@playwright/test';
+import { InscriptionView } from '../pageobjects/InscriptionView';
+import type { Section } from '../pageobjects/Section';
+import { screenshot } from './screenshot-util';
+
+const GENERIC_PID = {
+  SCRIPT: '168F0C6DF682858E-f3',
+  USER_TASK: '168F0C6DF682858E-f5',
+  DATABASE: '18141E75C9CEDD35-f3'
+} as const;
+
+test.describe('Browsers', () => {
+  test('ScriptingBrowser', async ({ page }) => {
+    await openSection(page, GENERIC_PID.SCRIPT, 'Output', 'Code').then(async section => {
+      await section.scriptArea().openBrowsers();
+      const browser = section.scriptArea().browser();
+      const dialog = page.getByRole('dialog');
+
+      await browser.openTab('Attribute');
+      await expect(dialog.getByRole('row', { name: 'addPerson' }).first()).toBeVisible();
```

Comment:
shouldn't this be provided by the browser page object?
something like `browser.content` which returns the locator of the content.